### PR TITLE
Release v0.5.0

### PR DESCRIPTION
**Pull request by thanodnl**
This PR is in preparation of the release of ringpop-go v0.5.0.

The features implemented in this version are described in `CHANGES.md`

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
 ringpop-go changes
 ==================
 
+v0.5.0
+------
+
+* Feature: Automatic healing of fully partitioned rings that could be cause by
+  temporary network failure. #135 #128 #129
+* Fix: Fullsyncs are now executed in both directions to prevent persistent
+  asymetrical partitions. #134
```
Comment (dansimau) [inline CHANGES.md]: s/asymetrical/asymmetrical

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+
+It is advised to complete the deployment of this version within 24 hours after
+the first node is upgraded. Since the reaping of faulty nodes is added to this
+release a node that upgraded to this version will mark all the faulty members of
+the ringpop cluster as tombstones (the new state introduced for the reaping of
+faulty nodes) 24 hours after the deploy. If older versions of ringpop run in a
+cluster that starts declaring these `tombstones` the cluster will jump in endless
+fullsyncs mode. Ringpop operates normally under these conditions but extra load
+on both network and CPU are in effect during this time. The fullsyncs will
+automatically resolve as soon as all members are upgraded to this version.
```
Comment (Motiejus) [inline CHANGES.md]: What if one is downgrading? I assume full-syncs will happen too, but it would be useful to mention this explicitly.

**Comment by Motiejus** [comment]
LGTM

**Comment by CorgiMan** [comment]
Looks good

**Pull request merged**